test/integration_test/migration_dr_actions_test.go

```diff
@@ -0,0 +1,548 @@
+//go:build integrationtest
+// +build integrationtest
+
+package integrationtest
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/libopenstorage/stork/pkg/log"
+	"github.com/libopenstorage/stork/pkg/storkctl"
+	"github.com/portworx/sched-ops/k8s/apps"
+	storkops "github.com/portworx/sched-ops/k8s/stork"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestDRActions(t *testing.T) {
+	// reset mock time before running any tests
+	err := setMockTime(nil)
+	log.FailOnError(t, err, "Error resetting mock time")
+	currentTestSuite = t.Name()
+
+	err = setSourceKubeConfig()
+	log.FailOnError(t, err, "failed to set kubeconfig to source cluster: %v", err)
+
+	log.InfoD("Using stork volume driver: %s", volumeDriverName)
+	log.InfoD("Backup path being used: %s", backupLocationPath)
+	setDefaultsForBackup(t)
+
+	// get the destination kubeconfig from configmap in source cluster so that it can be passed to storkctl commands
+	// since both the dr cli commands run in destination cluster
+	destinationKubeConfigPath, err = getDestinationKubeConfigFile()
+	log.FailOnError(t, err, "Error getting destination kubeconfig file")
+
+	t.Run("testDRActionFailoverMultipleNamespacesTest", testDRActionFailoverMultipleNamespacesTest)
+	t.Run("testDRActionFailoverSubsetNamespacesTest", testDRActionFailoverSubsetNamespacesTest)
+	t.Run("testDRActionFailoverWithMigrationRunningTest", testDRActionFailoverWithMigrationRunningTest)
+
+}
+
+// testDRActionFailoverMultipleNamespacesTest tests failover action for multiple namespaces
+func testDRActionFailoverMultipleNamespacesTest(t *testing.T) {
+	var testrailID, testResult = 87163548, testResultFail
```

Comment:
Are the testrail IDs correct?
According to https://portworx.testrail.net/index.php?/cases/view/297512
the ID for this case should be 297512 right ??